src/Domain/Creators/Sphere.hpp

```diff
@@ -38,15 +39,77 @@ class CoordinateMap;
 
-namespace domain {
-namespace creators {
-/// Create a 3D Domain in the shape of a sphere consisting of six wedges
-/// and a central cube. For an image showing how the wedges are aligned in
-/// this Domain, see the documentation for Shell.
-///
-/// The inner cube is a BulgedCube except if the inner cube sphericity is
-/// exactly 0. Then an Equiangular or Affine map is used (depending on if it's
-/// equiangular or not) to avoid a root find in the BulgedCube map.
+namespace domain::creators {
+
+namespace detail {
+
+/// Options for excising the interior of the sphere
+struct Excision {
+  static constexpr Options::String help = {
+      "Excise the interior of the sphere, leaving a spherical shell."};
+  template <typename BoundaryConditionsBase>
+  struct BoundaryCondition {
+    static std::string name() { return "ExciseWithBoundaryCondition"; }
+    using type = std::unique_ptr<BoundaryConditionsBase>;
+    static constexpr Options::String help = {
+        "The boundary condition to impose on the excision surface."};
+  };
+  struct NoBoundaryCondition {
+    static std::string name() { return "Excise"; }
+    using type = bool;
+    static constexpr Options::String help = {
+        "Specify 'True' to excise the interior of the Sphere. Instead of "
+        "specifying 'False', replace with the 'FillWithSphericity' option."};
+  };
+  template <typename Metavariables>
+  using options = tmpl::list<tmpl::conditional_t<
+      domain::BoundaryConditions::has_boundary_conditions_base_v<
+          typename Metavariables::system>,
+      BoundaryCondition<
+          domain::BoundaryConditions::get_boundary_conditions_base<
+              typename Metavariables::system>>,
+      NoBoundaryCondition>>;
+  Excision() = default;
+  Excision(std::unique_ptr<domain::BoundaryConditions::BoundaryCondition>
+               local_boundary_condition);
+  Excision(bool excise, const Options::Context& context = {});
+  std::unique_ptr<domain::BoundaryConditions::BoundaryCondition>
+      boundary_condition = nullptr;
+};
+
+/// Options for filling the interior of the sphere with a cube
+struct InnerCube {
+  static constexpr Options::String help = {
+      "Fill the interior of the sphere with a cube."};
+  struct Sphericity {
+    static std::string name() { return "FillWithSphericity"; }
+    using type = double;
+    static constexpr Options::String help = {
+        "Sphericity of the inner cube. A sphericity of 0 uses a product "
+        "of "
+        "1D maps as the map in the center. A sphericity > 0 uses a "
+        "BulgedCube. A sphericity of exactly 1 is not allowed. See "
+        "BulgedCube docs for why."};
+    static double lower_bound() { return 0.0; }
+    static double upper_bound() { return 1.0; }
+  };
+  using options = tmpl::list<Sphericity>;
+  double sphericity = std::numeric_limits<double>::signaling_NaN();
+};
+
+}  // namespace detail
+
+/*!
+ * \brief Create a 3D Domain in the shape of a sphere consisting of six wedges
```

Comment:
`consisting of spherical layers of Blocks (each layer collectively referred to as a shell) around a central cube`

Reply:
I just copied the option the option help string here